### PR TITLE
use set_service

### PR DESCRIPTION
**Pull request by dormi**

### CODE REVIEW

File: lib/dradis/plugins/nmap/importer.rb [hunk 1]
```diff
@@ -41,19 +41,30 @@ def import(params={})
...
-              state: port.state.to_s,
-              reason: port.reason,
-              name: port.try('service').try('name'),
-              product: port.try('service').try('product'),
-              scripts: port.try('scripts'),
-              version: port.try('service').try('version')
-          })
+          extra = []
+          if port.try('scripts')
+            port.try('scripts').map do |k, v|
```
Comment (aapomm) [inline lib/dradis/plugins/nmap/importer.rb]: There's no need to `try` the second time. If `port.try('scripts')` is successful, then `port.scripts` should be good.

File: lib/dradis/plugins/nmap/importer.rb [hunk 2]
```diff
@@ -41,19 +41,30 @@ def import(params={})
...
-              protocol: port.protocol.to_s,
-              state: port.state.to_s,
-              reason: port.reason,
-              name: port.try('service').try('name'),
-              product: port.try('service').try('product'),
-              scripts: port.try('scripts'),
-              version: port.try('service').try('version')
-          })
+          extra = []
+          if port.try('scripts')
```
Comment (aapomm) [inline lib/dradis/plugins/nmap/importer.rb]: Will symbols work here? `port.try(:scripts)`

File: lib/dradis/plugins/nmap/importer.rb [hunk 3]
```diff
@@ -41,19 +41,30 @@ def import(params={})
...
+              extra << { source: 'Nmap NSE scripts', id: k, output: v }
+            end
+          end
+          host_node.set_property(
+            :services,
+            port: port.number,
+            protocol: port.protocol.to_s,
+            state: port.state.to_s,
+            reason: port.reason,
+            name: port.try('service').try('name'),
```
Comment (aapomm) [inline lib/dradis/plugins/nmap/importer.rb]: This code would fail if `try(:service)` is nil.

**Review by aapomm — CHANGES_REQUESTED** [review]

File: lib/dradis/plugins/nmap/importer.rb (hunk 3 above)
Comment (dormi) [reply lib/dradis/plugins/nmap/importer.rb]: I think it works ? 
```
> nil.try('name')
=> nil
```

**Review by aapomm — APPROVED** [review]

**Pull request merged**